plugins/inputs/snmp_trap/snmp_trap.go

```diff
@@ -104,6 +103,9 @@ func (s *SnmpTrap) Init() error {
 			return err
 		}
 	case "netsnmp":
+		if s.Timeout == 0 {
+			s.Timeout = defaultTimeout
+		}
```

Comment:
Why setting it if it's an unused option?

Reply:
I agree. However, I'm not sure anymore the parameter is actually unused... It is used with the netsnmp translator isn't it?

Reply:
How/where?

Reply:
@Hipska the timeout is [passed to the netsnmp translator](https://github.com/influxdata/telegraf/blob/d075815f29d73a7fb880c11d24a1b3a4901cc520/plugins/inputs/snmp_trap/snmp_trap.go#L107) which in turn uses it for the [command execution](https://github.com/influxdata/telegraf/blob/d075815f29d73a7fb880c11d24a1b3a4901cc520/plugins/inputs/snmp_trap/netsnmp.go#L62).